kore/src/Kore/Equation/Sentence.hs

```diff
@@ -60,6 +64,16 @@ data MatchEquationError variable
     deriving anyclass (SOP.Generic, SOP.HasDatatypeInfo)
     deriving anyclass (Debug)
 
+instance InternalVariable variable => Pretty (MatchEquationError variable) where
+    pretty (NotEquation term) = "The given term is not an equation:\n" <> unparse term
```

Comment:
Use `Pretty.vsep` so that indentation is handled properly:

```suggestion
    pretty (NotEquation term) = Pretty.vsep ["The given term is not an equation:", unparse term]
```